content/docs/getting-started.mdx

```diff
@@ -11,7 +11,11 @@ The easiest way to start is to use our React + Material UI seed app.
    ```bash
    npm ci
    ```
-3. Run the app with:
+3. Build the app with:
+   ```bash
+   npm run Build
+   ```
+4. Run the app with:
    ```
    npm start
    ```
```

Comment:
Shouldn't we change this rather to the development workflow? i.e.

~~~suggestion
3. Run the app with:
   ```
   npm run dev
   ```
~~~